ceph/rados/scheduled_scrub_scenarios.py

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
This is a test module. We already have one in `tests` directory. Do we need this?

Reply:
No it is not required. The file had mistakenly been added to the commit. I removed the file